From 142af02b825bc638427e4354a53ad61438db824f Mon Sep 17 00:00:00 2001
From: Micah Snyder <micasnyd@cisco.com>
Date: Tue, 30 May 2023 16:07:45 -0700
Subject: Freshclam: remove curl result warning

A warning printing the HTTP code and file size was accidentally
committed at the end of ClamAV 1.1.0 dev when fixing a bug.
Remove this warning.

Resolves: https://github.com/Cisco-Talos/clamav/issues/930
Patch-Name: Freshclam-remove-curl-result-warning.patch
---
 libfreshclam/libfreshclam_internal.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/libfreshclam/libfreshclam_internal.c b/libfreshclam/libfreshclam_internal.c
index e1ffeb1..c96b488 100644
--- a/libfreshclam/libfreshclam_internal.c
+++ b/libfreshclam/libfreshclam_internal.c
@@ -1300,7 +1300,6 @@ static fc_error_t downloadFile(
 
     /* Check HTTP code */
     curl_easy_getinfo(curl, CURLINFO_RESPONSE_CODE, &http_code);
-    logg(LOGG_WARNING, " ******* RESULT %ld, SIZE: %zu ******* \n", http_code, receivedFile.size);
     switch (http_code) {
         case 200:
         case 206: {
